### PR TITLE
Bug/fix stats reference

### PR DESCRIPTION
**Pull request by Brennan1994**

### CODE REVIEW

File: fda-model/paireddata/StepwisePairedData.cs [hunk 1]
```diff
@@ -93,11 +93,11 @@ public IPairedData multiply(IPairedData g)
 				        newYvals.Add(belowFragilityCurveValue);
 			        } else {
 				        //create a point on the curve just below the bottom of the levee at damage zero.
-				        newXvals.Add(bottom-.000000000001);
+				        newXvals.Add(bottom-.001);
```
Comment (HenryGeorgist) [inline fda-model/paireddata/StepwisePairedData.cs:96]: why do you decrease the value to .001?

File: fda-model/fda-model.csproj [hunk 2]
```diff
@@ -7,7 +7,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Statistics" Version="1.0.0" />
+    <PackageReference Include="USACE.HEC.FDA.Statistics" Version="1.0.0-Alpha" />
```
Comment (HenryGeorgist) [inline fda-model/fda-model.csproj:10]: great

**Review by HenryGeorgist — APPROVED** [review]

File: fda-model/paireddata/StepwisePairedData.cs (hunk 1 above)
Comment (Brennan1994) [reply fda-model/paireddata/StepwisePairedData.cs:96]: in Fragility curve units, those are feet. I figured a 1/1000th of a foot buffer was probably enough, and made it easier to write tests against it. Too hard to count how many zeroes were there when predicting the answers.

Comment (Brennan1994) [reply fda-model/paireddata/StepwisePairedData.cs:96]: Weird that it's in this pull request though. I may have gotten mixed up there. It is definitely unrelated to the stats bug.

**Pull request merged**